docs/software/programming.md

```diff
@@ -90,58 +90,39 @@ Go is used by some of the big organizations such as Google, BBC, Uber, Soundclou
 
     Checkout also the official tutorial [Get started with Go](https://golang.org/doc/tutorial/getting-started).
 
-=== "Install / Uninstall"
-
-    - We switched to automatic Go version detection. Here is an example:
-
-        ```sh
-        root@DietPi3:~# go version
-        go version go1.16.3 linux/arm
-        ```
-
-    - When the Go package gets uninstalled, the folder `/mnt/dietpi_userdata/go` is still kept.  
-      This is the place where packages are installed, custom compilations are run, sources are downloaded etc.
-
-        It is especially important to keep `/mnt/dietpi_userdata/go` as long as we don't have a good dependency system that blocks uninstalls of dependencies. Otherwise it would be possible to uninstall Go while [OpenBazaar](../social/#openbazaar) is still installed.  
-        As a side effect, removing `/mnt/dietpi_userdata/go` would mean also removing [OpenBazaar](../social/#openbazaar).
-
-=== "Directories"
-
-    - `/mnt/dietpi_userdata/go`: This is the place where packages are installed, custom compilations are run, sources are downloaded etc.  
-      This path is what is given in the environment variable GOPATH.
-
-        !!! note "GOPATH is a global setting"
-            In normal Go installations, GOPATH is a user specific environment variable. In DietPi it is global, i.e. all users have the same module cache and all see the same binaries below GOPATH/bin.
-
-    - `/usr/local/go`: This is the place where the Go package is installed.
-    - `/mnt/dietpi_userdata/go/pkg/mod`: This is the path for 3rd party Go packages. The environment variable GOMODCACHE directs to this.
-
-=== "Getting started"
-
     Some common Go commands are:
 
     - `go version`: Prints the installed Go version
     - `go env`: Prints the Go internal environment variables (e.g. GOPATH). Can also be used e.g. like `$(go env GOPATH)/bin`
     - `go mod tidy <MODULNAME>`: Generate a Go module
     - `go help`: Start the Go internal help in general, details for commands e.g. via `go help build`
 
+=== "Directories"
+
+    - `/usr/local/go`: This is the place where the Go development tools is installed.
+    - When installing packages via `go get` or `go install`, they are installed into the users home directory `~/go` by default. A custom path can be set via `GOPATH` environment variable. To also load binaries of compiled Go packages into your `PATH`, something like this can be done:
```

Comment:
I propose "A custom path can be set via the `GOPATH` environment variable."

Reply:
```suggestion
    - When installing packages via `go get` or `go install`, they are installed into the users home directory `~/go` by default. A custom path can be set via the `GOPATH` environment variable. To also load binaries of compiled Go packages into your `PATH`, something like this can be done:
```